project.yaml

```
@@ -30,6 +30,15 @@ actions:
       moderately_sensitive:
         measure: output/measures/pf_medications_measures.csv
 
+  generate_pf_statistics_measures:
+    run: >
+      ehrql:v1 generate-measures analysis/measures_definition_pf_statistics.py
```

just an idea for making the file name a bit clearer? what do you think?

```suggestion
      ehrql:v1 generate-measures analysis/measures_definition_pf_descriptive_stats.py
```